_data/sites.json

```
@@ -12526,7 +12526,15 @@
             "zenkit.com"
         ]
     },
-
+    {
```

```suggestion

    {
```
small suggestion just to keep the formatting of a line break between each entry.